build-system/tasks/helpers.js

```diff
@@ -536,7 +536,7 @@ const nameCache = {};
 const mangleIdentifier = {
   get(num) {
     const charset =
-      '$ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz0123456789';
+      'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ$_0123456789';
```

Comment:
Especially if we pursue this PR, then it makes sense to remove our stable id generation algo in favor of tersers default.